Userland/Libraries/LibJS/Runtime/ArrayBuffer.h

```diff
@@ -22,6 +22,11 @@ struct ClampedU8 {
 // 25.1.1 Notation (read-modify-write modification function), https://tc39.es/ecma262/#sec-arraybuffer-notation
 using ReadWriteModifyFunction = Function<ByteBuffer(ByteBuffer, ByteBuffer)>;
 
+enum class PreserveResizability {
+    Fixed,
+    Preserve
```

Comment:
Always match spec names as closely as possible

```suggestion
    FixedLength,
    PreserveResizability
```